README.md

```diff
@@ -68,6 +68,15 @@ General information through talks tthat maybe  useful on PyData (various confere
   * Enthought: https://www.youtube.com/user/EnthoughtMedia
   * Continuum Analytics: https://www.youtube.com/channel/UCND4vKhJssAtK8p1Blfj14Q
 
+## Jupyter extensions
+
+Jupyter has a wide ecosystem of extensions that can be used to extend the functionality. Some useful extensions for HEP data analysis are summarised here.
+
+| Name         | Use             |
+| ------------ | --------------- |
+| nbdime       | Simplifies diffing and merging of jupyter notebooks that are stored in version control. |
+| jupytext     | Splits notebooks into a `.ipynb` and `.py` file for easier version control and to allow them to be run as scripts idependently of jupyter. |
```

Comment:
Splits notebooks into a `.ipynb` and `.py` files … independently …

Can you fix these 2 things and merge with the previous commit?

Reply:
Doesn't Jupyter Notebook let you save directly as `.py` from the Web interface?  Does it use jupytext behind the scenes?